tasks/copy_to_archive_adapter/README.md

```diff
@@ -85,6 +85,9 @@ bash-4.2# bin/run_tests.sh
 Note that Bamboo will run this same script via the `bin/run_tests.sh` script found
 in the cumulus-orca base of the repo.
 
+To run integration test for **copy_to_archive_adapter**, run `bin/run_integration_tests.sh` script from the
+`tasks/copy_to_archive_adapter` directory. Make sure the object to be copied exists in the source bucket first before running the test.
+
```

Comment:
Might be worth adding comments about environment variables, but as long as the script documents them I'm fine.

Reply:
added more info on README.md